src/main/java/hudson/scm/SubversionChangeLogBuilder.java

```diff
@@ -101,7 +101,7 @@ public SubversionChangeLogBuilder(Run<?,?> build, FilePath workspace, @Nonnull S
         this.env = env;
     }
 
-    public boolean run(Collection<SubversionSCM.External> externals, Result changeLog) throws IOException, InterruptedException {
+    public boolean run(@Nonnull Map<String, List<SubversionSCM.External>> externalsMap, Result changeLog) throws IOException, InterruptedException {
```

Comment:
This introduces a binary incompatibility, but [it doesn't look like any other plugins call this method](https://github.com/search?utf8=%E2%9C%93&q=user%3Ajenkinsci+SubversionChangeLogBuilder&type=Code). Might be good to add `@Restricted(NoExternalUse.class)` to the class in a followup PR.

Reply:
I will manage it